plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/deployimage/IDeployImagePageModel.java

```diff
@@ -103,4 +103,9 @@
 	 * @return <code>true</code> if the initialization succeeded, <code>false</code> otherwise.
 	 */
 	boolean initializeContainerInfo();
+	
+	/**
+	 * Free any resource
+	 */
+	void release();
```

Comment:
@fbricon In eclipse development, isn't the pattern to use 'dispose'?  It looks like we are just using this method name because that's what the docker tooling lib uses.